### ISSUE TITLE
Not able to install apk's from apk installer in latest aftermath wizard on Kodi 18 32bit

### ISSUE THREAD

**Issue by bringyoutolife** [drinfernoo/plugin.program.aftermath#51]
This issue I am about to describe is in the official release of Kodi 18 32 bit with the latest  version of aftermath wizard installed. I tested on Firestick 4k, fire TV cube, and a moto g6 phone.

 An issue exists with not being able to install apks from the Apk Installer section. You are able to download them and the spmc install pop up image shows up but when you click ok it doesnt launch the installation screen, instead it takes you back to the dialog asking if you want to download that apk so you can't install any apk files through the wizard anymore. Please fix this issue 🙏

**Comment by drinfernoo** [comment]
This is a limitation of Kodi 18.0, due to a change in Android's permissions system that has disallowed apps installing other apps. I'm still looking into a fix.

**Comment by bugatsinho** [comment]
It works though on kodi 17.6 in android 9 device!

**Comment by drinfernoo** [comment]
This is probably going to mean a change to Kodi in general, and I've opened a PR [here](https://github.com/xbmc/xbmc/pull/15424) to try and get the ball rolling.

**Comment by drinfernoo** [comment]
> It works though on kodi 17.6 in android 9 device!

Correct. The limitation comes as a side-effect of the Kodi Android app having `targetSdkVersion="26"`, which only happened as of Leia. Before that, they weren't required (or even able) to handle permissions the way Oreo does.

**Comment by bugatsinho** [comment]
> > It works though on kodi 17.6 in android 9 device!
> 
> Correct. The limitation comes as a side-effect of the Kodi Android app having `targetSdkVersion="26"`, which only happened as of Leia. Before that, they weren't required (or even able) to handle permissions the way Oreo does.

lets compile a fork!

**Comment by drinfernoo** [comment]
Until this is fixed on Kodi's end, 04cb0639be0889fc042675212f7ffeb90a8463c6 will disable the APK Installer section on Kodi versions greater than 18. I'm labeling this as wontfix until then.

**Comment by kiotronic** [comment]
Oh this is not good news... Not chance enable APK installing from Kodi 18 ?
